app/views/adjustments/show.html.erb

```diff
@@ -45,18 +45,20 @@
   <% @adjustment.line_items.each do |line_item|  %>
     <tr class="<%= line_item.quantity > 0 ? "positive" : "negative" %>">
       <td><%= (line_item.quantity > 0) ? "Added " : "Removed " %></td>
-      <td><%= line_item.quantity.abs %></td>
       <td><%= line_item.item.name %></td>
+      <td><%= line_item.quantity.abs %></td>
     </tr>
   <% end %>
     </table>
   <!-- /.box-body -->
-  <p>
 
   </div>
       <!-- /.box -->
-  <strong>Comments:</strong>
-  <%= @adjustment.comment %>
-</p>
+    <p>
```

Comment:
I'm worried that this file change is out of scope for this issue. Is there a reason it was needed?

Reply:
No reason, it's just that `<p>` is at wrong position. So fixed it. Came across this while creating `audit.html.erb`. Also the `</section>` tag is also not present in this file. So added it. Should I revert this back?

Reply:
Also the positions of `Item name` and `Item Quantity` in `table header` and `table body` are different

Reply:
Discussed with @armahillo . Resolving.